corehq/apps/locations/adjacencylist.py

```diff
@@ -110,11 +115,16 @@ def get_descendants(self, node, include_self=False):
             where = Q(parent_id=node.id)
 
         def make_cte_query(cte):
+            try:
+                base_queryset = self.filter(domain=node.domain).order_by()
+            except AttributeError:
+                base_queryset = self.all().order_by()
```

Comment:
It's been a while since I've looked at this - are you sure `node` always has a domain?  Might also want to write this as `if hasattr(node, 'domain')...` rather than catching an `AttributeError`, since that's rather broad (although it's all library code).